featuretools/primitives/standard/transform/time_series/rolling_outlier_count.py

```diff
@@ -101,8 +101,9 @@ def get_outliers_count(self, numeric_series):
         # type inference could not infer a numeric type
         if not len(numeric_series.dropna()):
             return np.nan
-        ww_series = init_series(numeric_series, logical_type="Double")
-        box_plot_info = ww_series.ww.box_plot_dict()
+        if hasattr(numeric_series, "ww"):
```

Comment:
You can check if WW is initialized directly by checking if the schema is None:
``` if numeric_series.ww.schema is None:```

If WW is not initialized the schema will be `None`, otherwise it will be a `ColumnSchema` object.